react/PartnerSites/Locales/Locales.js

```diff
@@ -15,46 +15,41 @@ export default function Locales({ locale, linkRenderer }) {
   });
 
   return (
-    <nav aria-labelledby="Locales" role="navigation" className={styles.root}>
-
+    <nav aria-labelledby='Locales' role='navigation' className={styles.root}>
       <ScreenReaderOnly>
-        <h1 id="Locales">Select your country</h1>
+        <h1 id='Locales'>Select your country</h1>
```

Comment:
We normally use double quotes for html/jsx, not sure if there is a prettier config to solve here.